syntax = "proto3";
option cc_enable_arenas = true;

service OptionChain {
  rpc GetOptionChain(OptionChainRequest) returns (OptionChainResponse) {}
}

message OptionChainRequest {
  string symbol = 1;
}

message OptionChainResponse {
  string symbol = 1;
  string status = 2;
}